recipes/tensorflowonspark/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "tensorflowonspark" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4d7a9bd1784112518e1bd49f02db16cb5d8a32c473e6114d89c2f3304da14db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - tensorflowonspark
+
+about:
+  home: "https://github.com/yahoo/TensorFlowOnSpark"
+  license: Apache Software
```

We prefer SPDX identifiers:
```suggestion
  license: Apache-2.0
```